src/adapters.ts

```diff
@@ -357,14 +363,24 @@ export function AdapterL1<TBase extends Constructor<TxSender>>(Base: TBase) {
                 tx.gasPerPubdataByte,
             );
 
+            const baseTokenAddress = await zksyncContract.baseTokenAddress();
+
             if (token == ETH_ADDRESS) {
                 checkBridgeWETHAllowed();
                 overrides.value ??= baseCost + BigInt(operatorTip) + BigInt(amount);
-
                 return {
                     contractAddress: to,
                     calldata: "0x",
-                    l2Value: amount,
+                    l2Value: 0,
```

Comment:
maybe we don't  need to update logic in the eth case because our chain doesn't use it. Keeps code less different with the upstream?